cincinnati/src/plugins/internal/cincinnati_graph_fetch.rs

```diff
@@ -0,0 +1,308 @@
+//! Plugin which implements fetching a Cincinnati graph via HTTP from a `/v1/graph`-compliant endpoint.
+//!
+//! This plugin is special so that it is expected to be the first plugin in the
```

Comment:
This is not strictly true, right?
Anything doing client-parameters validation/manipulation should be happening before this, I guess.

Reply:
Indeed. I removed that part of the description.